### PR TITLE
[Docs] Steps to compile mmcv-full on MLU machine

### PR DESCRIPTION
**Pull request by chentianyi1**
## Motivation

Added MMCV Installation guide on MLU devices

## Modification

Modified mmcv/docs/en/get_started/build.md and mmcv/docs/zh_cn/get_started/build.md

### CODE REVIEW

**Comment by GuWei007** [comment]
@zhouzaida   please review

File: docs/zh_cn/get_started/build.md [hunk 1]
```diff
@@ -389,3 +389,44 @@ w = torch.ones(10).float().npu()
...
+进入docker, [编译 MMCV MLU](#编译mmcv-mlu) 并[进行验证](#验证是否成功安装).
+
+#### 选项2：基于cambricon pytorch源码编译安装
+
+#### 安装CATCH:
+
+- 请向 service@cambricon.com 发送邮件或联系Cambricon工程师以获取合适版本的CATCH软件包，在您获得合适版本的CATCH软件包后，请参照 ${CATCH-path}/CONTRIBUTING.md 中的步骤安装CATCH.
+
+
+#### 编译MMCV MLU:
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:411]:
```suggestion
#### 编译 MMCV
```

File: docs/zh_cn/get_started/build.md [hunk 2]
```diff
@@ -389,3 +389,44 @@ w = torch.ones(10).float().npu()
...
+
+```bash
+git clone https://github.com/open-mmlab/mmcv.git -b master
+cd mmcv
+export MMCV_WITH_OPS=ON
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:416]:
```suggestion
export MMCV_WITH_OPS=1
```

File: docs/zh_cn/get_started/build.md [hunk 3]
```diff
@@ -389,3 +389,44 @@ w = torch.ones(10).float().npu()
...
+export FORCE_MLU=1
+python setup.py install
+```
+
+#### 验证是否成功安装
+完成上述安装步骤之后，您可以尝试运行下面的python代码以测试您是否成功在MLU设备上安装了mmcv-full:
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:422]:
```suggestion

完成上述安装步骤之后，您可以尝试运行下面的 Python 代码以测试您是否成功在 MLU 设备上安装了 mmcv-full:

```

File: docs/zh_cn/get_started/build.md [hunk 4]
```diff
@@ -389,3 +389,44 @@ w = torch.ones(10).float().npu()
...
+docker pull ${docker image}
+``` 
+
+进入docker, [编译 MMCV MLU](#编译mmcv-mlu) 并[进行验证](#验证是否成功安装).
+
+#### 选项2：基于cambricon pytorch源码编译安装
+
+#### 安装CATCH:
+
+- 请向 service@cambricon.com 发送邮件或联系Cambricon工程师以获取合适版本的CATCH软件包，在您获得合适版本的CATCH软件包后，请参照 ${CATCH-path}/CONTRIBUTING.md 中的步骤安装CATCH.
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:408]:
```suggestion
请向 service@cambricon.com 发送邮件或联系 Cambricon 工程师以获取合适版本的 CATCH 软件包，在您获得合适版本的 CATCH 软件包后，请参照 ${CATCH-path}/CONTRIBUTING.md 中的步骤安装 CATCH。
```

File: docs/zh_cn/get_started/build.md [hunk 5]
```diff
@@ -389,3 +389,44 @@ w = torch.ones(10).float().npu()
...
+首先请下载并且拉取寒武纪docker (请向 service@cambricon.com发邮件以获得最新的寒武纪pytorch发布docker)
+```
+docker pull ${docker image}
+``` 
+
+进入docker, [编译 MMCV MLU](#编译mmcv-mlu) 并[进行验证](#验证是否成功安装).
+
+#### 选项2：基于cambricon pytorch源码编译安装
+
+#### 安装CATCH:
```
[the remark this answers is not in the document]
Comment (chentianyi1) [reply docs/zh_cn/get_started/build.md]: fixed

File: docs/zh_cn/get_started/build.md [hunk 6]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
 output = softmax_focal_loss(x, y, 2.0, 0.25, w, 'none')
 print(output)
 ```
+
+### 在寒武纪 MLU 机器编译 mmcv-full
+
+#### 选项1: 基于寒武纪docker image 安装
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:395]:
```suggestion
#### 选项1: 基于寒武纪 docker image 安装
```

File: docs/zh_cn/get_started/build.md [hunk 7]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
...
+
+首先请下载并且拉取寒武纪docker (请向 service@cambricon.com发邮件以获得最新的寒武纪pytorch发布docker)
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:397]:
```suggestion
首先请下载并且拉取寒武纪 docker（请向 service@cambricon.com 发邮件以获得最新的寒武纪 pytorch 发布 docker）
```

File: docs/zh_cn/get_started/build.md [hunk 8]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
...
+```
+docker pull ${docker image}
+``` 
+
+进入docker, [编译 MMCV MLU](#编译mmcv-mlu) 并[进行验证](#验证是否成功安装).
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:402]:
```suggestion
进入 docker, [编译 MMCV MLU](#编译mmcv-mlu) 并[进行验证](#验证是否成功安装).
```

File: docs/zh_cn/get_started/build.md [hunk 9]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
...
+
+#### 选项2：基于cambricon pytorch源码编译安装
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:404]:
```suggestion
#### 选项2：基于 cambricon pytorch 源码编译安装
```

File: docs/zh_cn/get_started/build.md [hunk 10]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
...
+
+```bash
+git clone https://github.com/open-mmlab/mmcv.git -b master
+cd mmcv
+export MMCV_WITH_OPS=1
+export FORCE_MLU=1
+python setup.py install
+```
+
+#### 验证是否成功安装
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:420]:
```suggestion
#### 验证是否成功安装

```

File: docs/zh_cn/get_started/build.md [hunk 11]
```diff
@@ -389,3 +389,43 @@ w = torch.ones(10).float().npu()
...
+完成上述安装步骤之后，您可以尝试运行下面的 Python 代码以测试您是否成功在 MLU 设备上安装了 mmcv-full:
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/build.md:421]:
```suggestion
完成上述安装步骤之后，您可以尝试运行下面的 Python 代码以测试您是否成功在 MLU 设备上安装了 mmcv-full:

```

**Comment by OpenMMLab-Assistant001** [comment]
Hi @chentianyi1 ！We are grateful for your efforts in helping improve this open-source project during your personal time. 

Welcome to join OpenMMLab Special Interest Group (SIG) private channel on **Discord**, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "**mmsig + Github ID**" as a remark when adding friends：）

Thank you again for your contribution❤

**Pull request closed without merging**